website/.sailsrc

```diff
@@ -5,1798 +5,5 @@
   "_generatedWith": {
     "sails": "1.2.5",
     "sails-generate": "2.0.0"
-  },
```

Comment:
@mike-j-thomas I discarded the new changes, and also included a change that excludes the automatically-generated piece that seems to have been pushed up by accident in some other PR in the past.